lightning/config/global.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/BurntSushi/toml"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb-lightning/lightning/common"
+	"github.com/pingcap/tidb-lightning/lightning/log"
+)
+
+type GlobalLightning struct {
+	log.Config
+	StatusAddr string `toml:"status-addr" json:"status-addr"`
+	ServerMode bool   `toml:"server-mode" json:"server-mode"`
+
+	// The legacy alias for setting "status-addr"
+	PProfPort int `toml:"pprof-port" json:"-"`
```

Comment:
I think it's better to add some comments about the reason for ignoring the `PProfPort` field in the `JSON` configuration. Otherwise, we should not ignore it.